tests/acceptance/pageObjects/phoenixPage.js

```diff
@@ -162,6 +162,27 @@ module.exports = {
       for (const btn of cancelButtons) {
         await this.api.elementIdClick(btn.ELEMENT).waitForAnimationToFinish()
       }
+    },
+    browseToUserProfile: function () {
+      return this.click('@userMenuButton')
+    },
+    getUserProfileName: async function () {
+      let userInfo
+      await this
+        .waitForElementVisible('@profileInfoContainer')
+        .waitForElementVisible('@userProfileName')
+        .api.elements('@userProfileName', result => {
```

Comment:
Are you expecting to get more than one such ProfileNames

Reply:
done